test/util/functions_test.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
I didn't realize they were different, I just copy-pasted it from whatever file I happened to have open somewhere else :) I'll clean this up once I have a decision on the 3 options :/